modules/renter/hostdb/hostweight.go

```diff
@@ -162,20 +155,29 @@ func storageRemainingAdjustments(entry modules.HostDBEntry) float64 {
 	if entry.RemainingStorage < 200*requiredStorage {
 		base = base / 2 // 2x total penalty
 	}
-	if entry.RemainingStorage < 50*requiredStorage {
-		base = base / 3 // 6x total penalty
+	if entry.RemainingStorage < 150*requiredStorage {
+		base = base / 2 // 4x total penalty
+	}
+	if entry.RemainingStorage < 100*requiredStorage {
+		base = base / 3 // 12x total penalty
+	}
+	if entry.RemainingStorage < 80*requiredStorage {
+		base = base / 3 // 36x total penalty
+	}
+	if entry.RemainingStorage < 40*requiredStorage {
+		base = base / 4 // 144x total penalty
 	}
-	if entry.RemainingStorage < 25*requiredStorage {
-		base = base / 4 // 24x total penalty
+	if entry.RemainingStorage < 20*requiredStorage {
+		base = base / 5 // 720x total penalty
 	}
 	if entry.RemainingStorage < 10*requiredStorage {
-		base = base / 5 // 96x total penalty
+		base = base / 5 // 3,600x total penalty
 	}
 	if entry.RemainingStorage < 5*requiredStorage {
-		base = base / 10 // 960x total penalty
+		base = base / 5 // 14,400x total penalty
 	}
 	if entry.RemainingStorage < requiredStorage {
-		base = base / 10 // 9600x total penalty
+		base = base / 5 // 72,000x total penalty
```

Comment:
Penalties scale a lot harder now. This is particularly because the faster uploads means that the smaller hosts are actually filling up all the way, which is bad for renters.